packages/react-app/pages/profile/approval/index.js

```diff
@@ -72,6 +92,7 @@ function ApproveShareContactInformation() {
                 <Avatar
                   size={"xl"}
                   src={
+                    recruiter.avatar ||
                     "https://images.unsplash.com/photo-1520810627419-35e362c5dc07?ixlib=rb-1.2.1&q=80&fm=jpg&crop=faces&fit=crop&h=200&w=200&ixid=eyJhcHBfaWQiOjE3Nzg0fQ"
```

Comment:
Can we use a blockies library like: https://github.com/download13/blockies If integrating this will take time, don't worry about it and we can pick it up in the next sprint

Reply:
added blockies as placeholders :)